src/uhttp.c

```diff
@@ -307,9 +310,63 @@ static int process_header_line(const unsigned char* buffer, size_t len, size_t*
     return result;
 }
 
+static void log_data_line(HTTP_CLIENT_HANDLE_DATA* http_data, const char* text_line)
+{
+    char* authStart = strstr(text_line, AUTH_HEADER);
```

Comment:
So here, if Proxy-Authorization is before the Authorization header, authStart could be set to the middle of Proxy-Authorization. That is not the correct logic.

Reply:
Fixed.